conf/machine/include/qcom-base.inc

```diff
@@ -6,6 +6,9 @@ PREFERRED_PROVIDER_virtual/kernel ?= "linux-yocto-dev"
 KERNEL_IMAGETYPE ?= "Image"
 KERNEL_IMAGETYPES ?= "Image.gz"
 
+# For dtb.bin generation
+KERNEL_CLASSES:append = " linux-qcom-dtbbin "
```

Comment:
is it possible to use the `+=` instead of the `append` ?

Reply:
can you explain why you are making this suggestion? I don't see what the suggestion improves

Reply:
The `+=` is the preferred way to changes variables and applies during parsing but sometimes is not possible to do and in this cases we need to use the `append`. I am not now on my build machine but I will verify if can use the `+=`. Anyway this is not a big deal and my suggestion is more to follow the good practices I see upstream in oe/core.

Reply:
Yeah, += is better as you can easily replace and manipulate at other places by simply replacing the variable, append is harder to maintain as you require a remove, not only a set.

At the board level configuration we should avoid append.